.travis.yml

```diff
@@ -1,20 +1,21 @@
 language: python
 python:
-  - "3.6"
+  - "3.10"
+  - "3.11"
 install:
     - pip install --upgrade pip
     - pip install -r requirements.txt
     - pip install -r requirements_dev.txt
 env:
   global:
     - CC_TEST_REPORTER_ID=972a29fc370fae49a37220d1c23dafd0e71184217547376c7c495cf9670aa607
-    - CT_REGISTER_MODE=False
     - PORT=5037
     - API_VERSION=v1
     - LOCAL_URL=http://127.0.0.1:5037
-    - CT_URL=http://127.0.0.1:9000
-    - CT_TOKEN=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6Im1pY3Jvc2VydmljZSIsImNyZWF0ZWRBdCI6IjIwMTYtMDktMTQifQ.IRCIRm1nfIQTfda_Wb6Pg-341zhV8soAgzw7dd5HxxQ
-
+    - GATEWAY_URL=http://127.0.0.1:9000
+    - MICROSERVICE_TOKEN=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6Im1pY3Jvc2VydmljZSIsImNyZWF0ZWRBdCI6IjIwMTYtMDktMTQifQ.IRCIRm1nfIQTfda_Wb6Pg-341zhV8soAgzw7dd5HxxQ
```

Comment:
Just double-checking, it's okay for this to be public?

Reply:
yes